app/views/sponsors/show.json.jbuilder

```diff
@@ -1 +1,2 @@
 json.extract! @sponsor, :id, :name, :description, :active, :sensor_ids, :created_at, :updated_at
+json.logo_url image_url(@sponsor.logo_source || 'coredump.png')
```

Comment:
Does this mean that the logo will always be set to `coredump.png` if unset? In that case, it should probably be `null` instead.